datacatalog/pkg/manager/impl/artifact_manager.go

```diff
@@ -369,6 +379,10 @@ func (m *artifactManager) UpdateArtifact(ctx context.Context, request *datacatal
 
 	// update artifact in DB, also replaces/upserts associated artifact data
 	artifactModel.ArtifactData = artifactDataModels
+	logger.Debugf(ctx, "ArtifactModel is %+v", artifactModel)
+	logger.Debugf(ctx, "ArtifactModel ArtifactData is %+v", artifactModel.ArtifactData)
+	logger.Debugf(ctx, "ArtifactModel SerializedMetadata is %+v", artifactModel.SerializedMetadata)
```

Comment:
if we're going to leave this, can it be a single log line rather than 3? The first line should print the entire `artifactModel`, then the next two are printing fields that should already be exposed in the first.